Dockerfile.py3

```diff
@@ -15,7 +15,7 @@
 #     Use 'docker run' without '--rm' flag for keeping the container and use
 #     'docker commit <container hash> <new image>' to extend the original image
 
-FROM ubuntu:18.04
+FROM python:3.7-stretch
```

Comment:
I've always been puzzled about base image not clearly coming from a well known distrib and stuff.
What I mean here, is even though the Ubuntu base image is larger than some other ones, it makes it explicit the project compiles/run fine on Ubuntu.
Of course if changing the base image make a 50% time gain, then it can be discussed/justified. If it's a 10% then I'm not sure personally.
So this kinda of change also could be or not hard to make it to master sometimes even as single PR.
It deserves to be discussed, but I would rather have it on a dedicated PR since it could start some long discussions.

Reply:
good catch. I'm also in favor of sticking to the ubuntu base image, simply because it has a more widely known configuration